src/lib/schema.ts

```diff
@@ -1,158 +1,96 @@
-	expiresAt: datetime("expires_at").notNull()
+export const sessions = pgTable('sessions', {
+    id: text('id').primaryKey(),
+    userId: text('user_id')
+        .notNull()
+        .references(() => users.id),
+    expiresAt: timestamp('expires_at', {
+        withTimezone: true,
+        mode: 'date'
+    }).notNull()
 });
 
-export const resetToken = mysqlTable("password_reset_token", {
-	id: varchar("id", {
-		length: 64
-	}).primaryKey(),
-    user_id: varchar("user_id", {
-        length: 64
-    }),
+export const resetTokens = pgTable('password_reset_tokens', {
+    id: text('id').primaryKey(),
+    user_id: text('user_id'),
     expires: bigint('expires', {
         mode: 'number'
     })
 });
 
-export const verificationToken = mysqlTable("email_verification_token", {
-	id: varchar("id", {
-		length: 64
-	}).primaryKey(),
-    user_id: varchar("user_id", {
-        length: 64
-    }),
+export const verificationTokens = pgTable('email_verification_tokens', {
+    id: text('id').primaryKey(),
+    user_id: text('user_id'),
     expires: bigint('expires', {
         mode: 'number'
     })
 });
 
-export const events = mysqlTable("events", {
-    id: varchar("id", {
-		length: 64
-	}).primaryKey(),
-    event_name: varchar("event_name", {
-		length: 64
-	}),
-    event_description: varchar("event_description", {
-		length: 256
-	}),
-    event_start: int("event_start"),
-    event_end: int("event_end")
+export const events = pgTable('events', {
+    id: text('id').primaryKey(),
+    event_name: text('event_name'),
+    event_description: text('event_description'),
+    event_start: bigint('event_start', {
+        mode: 'number'
+    }),
+    event_end: bigint('event_end', {
+        mode: 'number'
+    })
 });
 
-export const challenges = mysqlTable("challenges", {
-    id: varchar("id", {
-		length: 64
-	}).primaryKey(),
-    event_id: varchar("event_id", {
-		length: 64
-	}).references(() => events.id).primaryKey(),
-    challenge_name: varchar("challenge_name", {
-		length: 64
-	}).unique(),
-    challenge_category: varchar("challenge_category", {
-		length: 64
-	}),
-    challenge_difficulty: varchar("challenge_difficulty", {
-		length: 64
-	}),
-    challenge_description: varchar("challenge_description", {
-		length: 256
-	}),
-    container_file: varchar("container_file", {
-		length: 128
-	}),
-    static_file_url: varchar("static_file_url", {
-		length: 256
-	}),
-    needs_container: boolean("needs_container"),
+export const challenges = pgTable('challenges', {
+    id: text('id').primaryKey(),
+    event_id: text('event_id')
+        .references(() => events.id),
+    challenge_name: text('challenge_name').unique(),
+    challenge_category: text('challenge_category'),
+    challenge_difficulty: text('challenge_difficulty'),
+    challenge_description: text('challenge_description'),
+    container_file: text('container_file'),
+    static_file_url: text('static_file_url'),
+    needs_container: boolean('needs_container')
 });
```

Comment:
Add hints property to challenges

Reply:
HInts might depend on Deployment API changes, not adding yet but noted.